base_layer/core/src/mining/blake_miner.rs

```diff
@@ -65,6 +65,7 @@ impl CpuBlakePow {
                     std::u64::MAX - last_measured_nonce + nonce
                 };
                 let hash_rate = hashes as f64 / start.elapsed().as_micros() as f64;
+                hashrate.store((hash_rate * 1000.0) as u64, Ordering::Relaxed);
```

Comment:
Why not * 1000000 and go straight to H? u64 is a huge number

Reply:
fair enough.  updating